.github/scripts/trymerge.py

```diff
@@ -929,52 +960,54 @@ def post_comment(msg: str) -> None:
 def prefix_with_github_url(suffix_str: str) -> str:
     return f"https://github.com/{suffix_str}"
 
-
-def check_for_sev(org: str, project: str, force: bool) -> None:
```

Comment:
did you mean to get rid of this function?

Reply:
no. i did not whoops